src/main/java/org/spongepowered/api/event/cause/EventContextKeys.java

```diff
@@ -22,32 +22,27 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.event.cause.entity.spawn;
-
-import org.spongepowered.api.Sponge;
-import org.spongepowered.api.entity.Entity;
-import org.spongepowered.api.world.weather.Weather;
+package org.spongepowered.api.event.cause;
```

Comment:
It could help inter-plugin consistency as well if they are cataloged. And then it can also impose constraints of what can be behind a context key (with the more generic one just being `?` if need be).